src/database.rs

```diff
@@ -189,6 +220,31 @@ impl<'a> Database<'a> {
         };
         String::from_utf8_lossy(cstr.to_bytes())
     }
+
+    /// Create a new collection in this database.
+    pub fn has_collection<S: Into<Vec<u8>>>(
```

Comment:
Function looks good, there's no test for it yet.

Reply:
done